Server/rainbow-server/Sources/Application/Routes/ScoreRoutes.swift

```diff
@@ -16,10 +16,9 @@ func initializeScoreRoutes(app: App) {
     client = app.services.couchDBService
     
     app.router.get("/entries", handler: getAllEntries)
-    app.router.get("/entries/", handler: getOneEntry)
+    app.router.get("/entries", handler: getOneEntry)
     app.router.post("/entries", handler: addNewEntry)
     app.router.put("/entries", handler: updateEntry)
-    app.router.delete("/entries", handler: deleteEntry)
```

Comment:
Are we sure we'll never need a DELETE route? @stevemart or @antonmc would love your opinion on this as well.

Reply:
I don't think we need to delete anything. We will update the same document everytime.

Reply:
Ok, we can always add it later if it comes up.

Reply:
i'm late to this, but yah, we don't necessarily need a delete handler right now.